src/cqrs/src/api/xxENSONOxx.xxSTACKSxx.Infrastructure.IntegrationTests/CosmosDb/Integration/IntegrationFixture.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Xunit;
+using xxENSONOxx.xxSTACKSxx.Infrastructure.Configuration;
+using xxENSONOxx.xxSTACKSxx.Infrastructure.IntegrationTests.CosmosDb.DataModel;
+using xxENSONOxx.xxSTACKSxx.Shared.Configuration;
+using Config = xxENSONOxx.xxSTACKSxx.Shared.Testing.Settings.Configuration;
+
+namespace xxENSONOxx.xxSTACKSxx.Infrastructure.IntegrationTests.CosmosDb.Integration;
+
+public class IntegrationFixture : IAsyncLifetime
+{
+    private CosmosClient CosmosClient;
+    private CosmosDbConfiguration Settings;
+    private SecretResolver _secretResolver;
+
+    public async Task InitializeAsync()
+    {
+        Settings = Config.For<CosmosDbConfiguration>("CosmosDB");
+        _secretResolver = new SecretResolver();
+
+        //Notes:
+        // if using an azure instance to run the tests, make sure you set the environment variable before you start visual studio
+        // Ex: CMD C:\> setx COSMOSDB_KEY ABCDEFGASD==
```

Comment:
this example caters to windows only

Reply:
I believe this is because Windows is the only operating system which has a Cosmos emulator at the moment and this is directed towards that.